astrokat/observe_main.py

```diff
@@ -477,7 +475,8 @@ def run_observation(opts, kat):
                         if not tgt:
                             break
                         # check enough time remaining to continue
-                        if time_remaining < tgt['duration']:
+                        if obs_duration > 0 and\
+                           time_remaining < tgt['duration']:
```

Comment:
Rather use parenthesis, as per our coding guidelines:
```python
if (obs_duration > 0 and
        time_remaining < tgt['duration']):
    done = True
    break
```
Or you could do it all on one line - that would fit within our 90 char limit.

Reply:
Added the parenthesis implementation , tx